tfac/dataHelpers.py

```diff
@@ -21,7 +21,7 @@ def importLINCSprotein():
 
 def ohsu_data():
     """ Import OHSU data for PARAFAC2"""
-    return pd.read_csv(join(path_here, "tfac/data/ohsu/MDD_RNAseq_Level4.csv"))
+    return pd.read_csv(join(path_here, "tfac/data/ohsu/RNAseq_Level4.txt"), delimiter="\t")
```

Comment:
What is the difference between these files?

Reply:
The WGCNA needs the data to be tab-delimited. I saved the data in a tab-delimited format and renamed it

Reply:
Aha, ok.